packages/nuekit/src/init.js

```diff
@@ -10,7 +10,8 @@ export async function init({ dist, is_dev, esbuild }) {
 
   // directories
   const cwd = process.cwd()
-  const srcdir = new URL('.', import.meta.url).pathname
+  const pathname = new URL('.', import.meta.url).pathname
+  const srcdir = pathname.startsWith('/') ? pathname.slice(1) : pathname
```

Comment:
<img width="470" alt="image" src="https://github.com/nuejs/nue/assets/6647633/c145be10-b83b-4e79-b923-81c8f0b056b2">

Seems this change can fail tests on non-Win OS